packages/model-viewer/src/features/ar.ts

```diff
@@ -229,7 +221,7 @@ export const ARMixin = <T extends Constructor<ModelViewerElementBase>>(
           await this[$enterARWithWebXR]();
           break;
         case ARMode.SCENE_VIEWER:
-          openSceneViewer(this.src!, this.alt || '', this.arScale);
+          openSceneViewer(this.src!, this.arScale);
```

Comment:
Why did we change this behavior?

I'm all for us making this customizable, but short of a clearly spelled out design goal (in the form of a filed issue) I would say this should remain unchanged.

Reply:
Okay, here's the spelled out design goal: #1143. Still open for discussion.

Reply:
I left some feedback. This probably needs more discussion before we can move on it, so for now I would recommend reverting the change.

Reply:
I registered my disagreement, but the change has been reverted here. I'm also not sure what existing query parameters will do in this new intent scheme. I think they won't work anymore, but maybe there's a way to fix that? I got into some trouble with `?` and `&`.